index.html

```diff
@@ -2223,14 +2226,47 @@ <h3>Presentations</h3>
 
         <p>
 The contents of the <code>verifiableCredential</code> <a>property</a> shown
-above are <a>verifiable credentials</a>, as described by this specification. The
-contents of the <code>proof</code> <a>property</a> are proofs, as described by
-the Data Integrity [[VC-DATA-INTEGRITY]] specification. An example of a
-<a>verifiable presentation</a> using the JWT proof mechanism is
+above are <a>verifiable credential</a> graphs, as described by this
+specification. The contents of the <code>proof</code> <a>property</a> are proof
+graphs, as described by the Data Integrity [[VC-DATA-INTEGRITY]] specification.
+An example of a <a>verifiable presentation</a> using the JWT proof mechanism is
 provided in the Securing Verifiable Credentials using JOSE and COSE
 [[?VC-JOSE-COSE]] specification.
         </p>
 
+        <section>
+          <h4>Verifiable Credential Graphs</h4>
+
+          <p>
+When expressing <a>verifiable credentials</a> in a <a>presentation</a>, it is
+important to ensure that data in one <a>verifiable credential</a> is not
+mistaken to be the same data in another <a>verifiable credential</a>. For
+example, if one has two <a>verifiable credentials</a>, each containing an
+object of the following form: `{"type": "Person", "name": "Jane Doe"}`, it is
+not possible to tell if one object is describing the same person as the other
+object. In other words, prematurely merging data between two
+<a>verifiable credentials</a> can lead to a corrupted data model.
```

Comment:
I think I know but I do not exactly know what "prematurely merging" and "corrupted data model" supposed to mean. please paraphrase/elaborate.

Reply:
Fixed via https://github.com/w3c/vc-data-model/pull/1280/files#r1325644063.